doc/api/next_api_changes/2019-04-05-AL.rst

```diff
@@ -0,0 +1,4 @@
+`~pyplot.imsave` gained support for the ``metadata`` and ``pil_kwargs`` parameters
```

Comment:
I'd say this belongs in what's new instead of API changes. Otherwise looks good.

Reply:
sure, changed